app/jobs/delete_resources_job.rb

```diff
@@ -17,6 +17,8 @@ def perform(url, exhibit)
   private
 
   def remove_resource(json:, exhibit:)
+    return if json.empty?
```

Comment:
I wonder if you'd be open to removing empty lines in the body-parsing code on line 10:

```ruby
resource = resp.body.split("\n").reject(&:empty?)
```

That would prevent having to check later.